### DISCUSSION TITLE
如何让词云的tooltip显示对应的value而不是x和y？

### DISCUSSION THREAD

**Discussion by D15OS** [antvis/G2#6227] [Q&A] [answered]
现在词云的tooltip默认显示的是词的xy刻度，但我想让它显示数据中词对应的value，我应该如何配置？
在文档和demo中并没有找到对应的配置项，但在G2plot的demo中是可以实现这种效果的。

**Comment by D15OS** [comment, marked as answer]
找到了，添加以下配置即可
`tooltip: {
  items: ['value']
}`

**Reply by hustcc** [reply]
要不来一个 PR，把官网的词云图 tooltip 优化一下吧？

**Reply by D15OS** [reply]
我有空就来
